src/auro-popover.js

```diff
@@ -16,17 +16,22 @@ import Popover from "./popover";
  *
  * @attr {String} placement - Expects top/bottom - position for popover in relation to the element.
  * @attr {String} for - Defines an `id` for an element in the DOM to trigger on hover/blur.
+ * @attr {boolean} isPopoverVisible - Boolean for if popover is visible or not.
+ * @attr {boolean} isSticky - If true, when user clicks trigger, the popover will persist its visibility. If false, popover will disappear when mouseout over trigger.
```

Comment:
This is incorrect. The property a user will use is `sticky`. That is what you are using in your demo.

Reply:
Changed.